lib/bigtable/cache/redisClient.js

```diff
@@ -136,4 +171,15 @@ class RedisClient {
 
 }
 
-module.exports = RedisClient;
\ No newline at end of file
+const redisClient = {
```

Comment:
there is no real clojure here, so using this is ugly, suggestion: redisClient._instance, additionally i would straight away export the create function only